### PR TITLE
KM Update Slide-seq & Optimus Docs

### PR DESCRIPTION
**Pull request by kayleemathews**
### Description

Created the Slide-seq Count Matrix Overview doc for the website that contains descriptions of the global, column, and row attributes for the Slide-seq Loom output file. I also updated the Slide-seq Overview doc to point to the new count matrix doc and added the RRID information to the README.

In addition, metrics were removed from the Optimus Cell and Gene metrics CSV files, so I updated the Loom_schema.md doc to reflect those changes.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Review by kevinpalis — APPROVED** [review]
Looks great!

File: website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md [hunk 1]
```diff
@@ -37,39 +37,37 @@ The global attributes in the Loom apply to the whole file, not any specific part
...
-|`n_genes`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of genes detected by this cell. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.CellMetrics.n_genes)|
-|`genes_detected_multiple_observations`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of genes that are observed by more than one read in this cell. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.CellMetrics.genes_detected_multiple_observations)|
-| `reads_unmapped`| [SC Tools](https://github.com/HumanCellAtlas/sctools/blob/master/src/sctools/metrics/aggregator.py) | Reads that are non-transcriptomic |
+|`n_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads associated with this entity. |
```
Comment (ekiernan) [inline website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md:43]:
```suggestion
|`n_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads associated with this entity. n_reads, like all metrics, are calculated from the Optimus output BAM. Prior to alignment with STARsolo, reads are checked against the whitelist (1 hamming distance). These CB-corrected reads are the input to the STAR aligner. Then, the reads also get CB correction during STAR. For this reason, almost all reads in the aligned BAM have a CB tag and UB tag. Therefore,
 n_reads represents CB corrected reads, not all reads in the input FASTQ files. |

File: website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md [hunk 2]
```diff
@@ -37,39 +37,37 @@ The global attributes in the Loom apply to the whole file, not any specific part
...
+|`genomic_reads_fraction_bases_quality_above_30_mean`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The average fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity (included for 10x Cell Ranger count comparison). |
+|`genomic_reads_fraction_bases_quality_above_30_variance`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The variance in the fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity (included for 10x Cell Ranger count comparison). |
+|`genomic_read_quality_mean`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Average quality of Illumina base calls in the genomic reads corresponding to this entity. |
+|`genomic_read_quality_variance`|[warp-tools](https://github.com/broadinstitute/warp-tools)|Variance in quality of Illumina base calls in the genomic reads corresponding to this entity. |
+|`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity. See README.md for the definition of a Molecule. |
```
Comment (ekiernan) [inline website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md:59]:
```suggestion
|`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity (only reflects reads with CB and UB tags. |

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+| `input_id` | The `input_id` provided to the pipeline as input and listed in the pipeline configuration file. This can be any string, but it's recommended for this to be consistent with any sample metadata. |
+| `molecule_barcode_fraction_bases_above_30_mean` | The average fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity. |
+| `molecule_barcode_fraction_bases_above_30_variance` | The variance in the fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity.|
+|`molecules_with_single_read_evidence`| The number of molecules associated with this entity that are observed by only one read. |
+|`n_fragments`| Number of fragments corresponding to this entity. |
+| `n_genes` | The number of genes detected by this bead. |
+|`n_mitochondrial_genes`| The number of mitochondrial genes detected by this bead. |
+|`n_mitochondrial_molecules`| The number of molecules from mitochondrial genes detected for this bead. |
+|`n_molecules`| Number of molecules corresponding to this entity. |
+|`n_reads`| The number of reads associated with this entity. |
```
Comment (ekiernan) [inline website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:57]:
```suggestion
|`n_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads associated with this entity. n_reads, like all metrics, are calculated from the Optimus output BAM. Prior to alignment with STARsolo, reads are checked against the whitelist (1 hamming distance). These CB-corrected reads are the input to the STAR aligner. Then, the reads also get CB correction during STAR. For this reason, almost all reads in the aligned BAM have a CB tag and UB tag. Therefore, n_reads represents CB corrected reads, not all reads in the input FASTQ files. |

File: website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md [hunk 4]
```diff
@@ -84,31 +82,25 @@ The global attributes in the Loom apply to the whole file, not any specific part
...
+|`genomic_reads_fraction_bases_quality_above_30_mean`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The average fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity (included for 10x Cell Ranger count comparison). |
+|`genomic_reads_fraction_bases_quality_above_30_variance`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The variance in the fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity (included for 10x Cell Ranger count comparison). |
+|`genomic_read_quality_mean`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Average quality of Illumina base calls in the genomic reads corresponding to this entity. |
+|`genomic_read_quality_variance`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Variance in quality of Illumina base calls in the genomic reads corresponding to this entity. |
+|`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity. See README.md for the definition of a Molecule. |
```
Comment (ekiernan) [inline website/docs/Pipelines/Optimus_Pipeline/Loom_schema.md:98]: |`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity (only reflects reads with CB and UB tags. |

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+| `input_id` | The `input_id` provided to the pipeline as input and listed in the pipeline configuration file. This can be any string, but it's recommended for this to be consistent with any sample metadata. |
+| `molecule_barcode_fraction_bases_above_30_mean` | The average fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity. |
+| `molecule_barcode_fraction_bases_above_30_variance` | The variance in the fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity.|
+|`molecules_with_single_read_evidence`| The number of molecules associated with this entity that are observed by only one read. |
+|`n_fragments`| Number of fragments corresponding to this entity. |
+| `n_genes` | The number of genes detected by this bead. |
+|`n_mitochondrial_genes`| The number of mitochondrial genes detected by this bead. |
+|`n_mitochondrial_molecules`| The number of molecules from mitochondrial genes detected for this bead. |
+|`n_molecules`| Number of molecules corresponding to this entity. |
```
Comment (ekiernan) [inline website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:56]:
```suggestion
|`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity (only reflects reads with CB and UB tags. |

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+|`genomic_read_quality_variance`| Variance in quality of base calls in the genomic reads corresponding to this entity. |
+|`genomic_reads_fraction_bases_quality_above_30_mean`| The average fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity. |
+|`genomic_reads_fraction_bases_quality_above_30_variance`| The variance in the fraction of bases in the genomic read that receive quality scores greater than 30 across the reads of this entity. |
+|`molecule_barcode_fraction_bases_above_30_mean`| The average fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity. |
+|`molecule_barcode_fraction_bases_above_30_variance`| The variance in the fraction of bases in molecule barcodes that receive quality scores greater than 30 across the reads of this entity. |
+|`molecules_with_single_read_evidence`| The number of molecules associated with this entity that are observed by only one read. |
+|`n_fragments`| Number of fragments corresponding to this entity. |
+|`n_molecules`| Number of molecules corresponding to this entity. |
```
Comment (ekiernan) [inline website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:89]:
```suggestion
|`n_molecules`|[warp-tools](https://github.com/broadinstitute/warp-tools)| Number of molecules corresponding to this entity (only reflects reads with CB and UB tags. |

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+|`n_reads`| The number of reads associated with this entity. |
```
Comment (ekiernan) [inline website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:90]:
```suggestion
|`n_reads`|[warp-tools](https://github.com/broadinstitute/warp-tools)| The number of reads associated with this entity. n_reads, like all metrics, are calculated from the Optimus output BAM. Prior to alignment with STARsolo, reads are checked against the whitelist (1 hamming distance). These CB-corrected reads are the input to the STAR aligner. Then, the reads also get CB correction during STAR. For this reason, almost all reads in the aligned BAM have a CB tag and UB tag. Therefore, n_reads represents CB corrected reads, not all reads in the input FASTQ files. |

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md (hunk 5 above)
Comment (kayleemathews) [reply website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:56]:
```suggestion
|`n_molecules`| Number of molecules corresponding to this entity (only reflects reads with CB and UB tags. |
```

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md (hunk 3 above)
Comment (kayleemathews) [reply website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:57]:
```suggestion
|`n_reads`| The number of reads associated with this entity. n_reads, like all metrics, are calculated from the Optimus output BAM. Prior to alignment with STARsolo, reads are checked against the whitelist (1 hamming distance). These CB-corrected reads are the input to the STAR aligner. Then, the reads also get CB correction during STAR. For this reason, almost all reads in the aligned BAM have a CB tag and UB tag. Therefore, n_reads represents CB corrected reads, not all reads in the input FASTQ files. |
```

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md (hunk 6 above)
Comment (kayleemathews) [reply website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:89]:
```suggestion
|`n_molecules`| Number of molecules corresponding to this entity (only reflects reads with CB and UB tags. |
```

File: website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md (hunk 7 above)
Comment (kayleemathews) [reply website/docs/Pipelines/SlideSeq_Pipeline/count-matrix-overview.md:90]:
```suggestion
|`n_reads`| The number of reads associated with this entity. n_reads, like all metrics, are calculated from the Optimus output BAM. Prior to alignment with STARsolo, reads are checked against the whitelist (1 hamming distance). These CB-corrected reads are the input to the STAR aligner. Then, the reads also get CB correction during STAR. For this reason, almost all reads in the aligned BAM have a CB tag and UB tag. Therefore, n_reads represents CB corrected reads, not all reads in the input FASTQ files. |
```

**Review by ekiernan — APPROVED** [review]
LGTM! Thank you, Kaylee!

**Pull request merged**